lumicks/pylake/scan.py

```diff
@@ -404,35 +407,47 @@ def _to_spatial(self, data):
         else:
             return data
 
-    def _plot(
+    @_deprecate_basescan_plot_args
+    def plot(
         self,
-        channel,
-        axes,
+        channel="rgb",
+        *,
         frame=0,
```

Comment:
Unfortunately, I found a counter example for the `Scan.plot(...)` deprecation path I'm afraid. 
```python
scan.plot("red", plt.gca(), 5)
```
Valid in the old API, raises `TypeError: plot() takes from 1 to 2 positional arguments but 4 were given` in the new one.